meshroom/core/node.py

```diff
@@ -750,6 +750,14 @@ def isSubmittedOrRunning(self):
                 return True
         return False
 
+    @Slot(result=str)
+    def getFirstChunkRunning(self):
```

Comment:
node.getFirstChunkRunning() => the name seems to indicate that it will return a chunk
Could be replaced by:
node.getStartDateTime() => the notion of chunk is not mentioned, but we can consider it an implementation detail